ScrapeRepos.js

```diff
@@ -0,0 +1,399 @@
+var __assign = (this && this.__assign) || function () {
```

Comment:
rewrite in TypeScript

Reply:
I just deleted it and kept the typescript version of the file